viewer/packages/tools/src/SmartOverlay/SmartOverlayTool.ts

```diff
@@ -35,7 +44,10 @@ export type SmartOverlayToolParameters = {
   defaultOverlayColor: THREE.Color;
 };
 
-export class SmartOverlayTool<MetadataType = DefaultMetadataType> extends Cognite3DViewerToolBase {
+/**
+ * Tool for adding and interacting with 2D overlays positioned at points in
+ */
+export class SmartOverlayTool<ContentType = DefaultOverlay3DContentType> extends Cognite3DViewerToolBase {
```

Comment:
I was also thinking that we can rename `SmartOverlayTool` into something like `Overlay3DTool` since now it is more flexible in the usage, so quite general name should be good.